src/adapters/ip_camera_adapter/ipcam.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python3
```

Comment:
Yeah - I wasn't sure where the best place to put it would be. In ipcam.py's case, it's an example of a client. Putting them in tools/scripts seems reasonable to me.

I'll also put a comment blobk at the top to identify what its for.